docs/nextflow-create-docker.md

```diff
@@ -98,7 +99,17 @@ In our example here, we will have you build your image using a `requirements.txt
 
 Our example will use the files in the [torch_cuda_test directory](https://github.com/uc-cdis/bio-nextflow/tree/master/nextflow_notebooks/containerized_gpu_workflows/torch_cuda_test) of the bio-nextflow repository. You can review the `readme` file in this directory for more information. It is a simple example that will build up from our base image by adding PyTorch. The Nextflow script will ultimately use a python script that checks the version of CUDA in the GPU instance and checks whether it is compatible with the version of PyTorch and CUDA available in the container.
 
-First, in the terminal, navigate to the directory where the downloaded Dockerfile and requirements.txt are located.
+First, download the contents of the [torch_cuda_test directory](https://github.com/uc-cdis/bio-nextflow/tree/master/nextflow_notebooks/containerized_gpu_workflows/torch_cuda_test). If you have `git`, the easiest way is to clone the repository:
```

Comment:
For this -- I put in the prerequisites that you should have already cloned the repo, so I would rather not put them here. I can add these steps up there, though

Reply:
gotcha, i still think "navigate to the directory where the downloaded Dockerfile and requirements.txt are located" is a bit out of the blue here, the prerequisite are a lot of text ago

maybe "from the directory where you cloned or downloaded the `bio-nextflow` repository as a prerequisite, navigate to the `torch_cuda_test` directory" and keep the `cd` command i added

Reply:
I updated this info to reflect your suggestions in a new PR